test/e2e/framework.go

```diff
@@ -563,7 +567,7 @@ func (data *TestData) deployAntreaFlowExporter(ipfixCollector string) error {
 	ac := []configChange{
 		{"FlowExporter", "true", true},
 		{"flowPollInterval", "\"1s\"", false},
-		{"activeFlowExportTimeout", "\"2s\"", false},
+		{"activeFlowExportTimeout", fmt.Sprintf("\"%v\"", exporterActiveFlowExportTimeout), false},
 		{"inactiveFlowExportTimeout", "\"1s\"", false},
```

Comment:
why not add const for inactiveFlowExportTimeout too?

Reply:
kk, done.